src/docker-client.ts

```diff
@@ -53,6 +53,15 @@ export type BindMount = {
 };
 type DockerodeBindMount = string;
 
+export type LogConfig = {
+  logDriver: string;
```

Comment:
I was going through the Docker docs to see what types we can add to this. It seems there's a limited number of logDrivers, but each one has different options. For example for Splunk, we would need to provide splunk-token and splunk-url (https://docs.docker.com/config/containers/logging/splunk/). As a result, what do you think about adding log-driver specific implementations? For example:

`withLogDriverForSplunk(token: String, url: String)`

I'm not sure what your use-case is, but you could add your specific log-driver implementation, and specify the mandatory options?

Reply:
Hm, I tend to disagree, as I think that people who need this feature also know quite well what log-driver they want and how the options they need for it are called, as they probably have this configuration already somewhere else.
On the other hand, I think it is quite possible that the use-case for most people is the same I had, which is that you configured a special log-driver on your docker host, but want to use the plain default 'json-file' logger with no special options for testcontainers. So what about having a withDefaultLogDriver() which has no more options and just resets the log driver to json-file with no options, which is the default in docker?

Reply:
@cristianrgreco any updates on this?